packages/cli/README.md

```diff
@@ -0,0 +1,84 @@
+# @cosmwasm/cli
+
+[![npm version](https://img.shields.io/npm/v/@cosmwasm/cli.svg)](https://www.npmjs.com/package/@cosmwasm/cli)
+
+## Installation and first run
+
+The `cosmwasm-cli` executable is available via npm. We recommend local
+installations to your demo project. If you don't have one yet, just
+`mkdir cosmwasm-cli-installation && cd cosmwasm-cli-installation && yarn init --yes`.
+
+### locally with yarn
+
+```
+$ yarn add @cosmwasm/cli --dev
+$ ./node_modules/.bin/cosmwasm-cli
+```
+
+### locally with npm
+
+```
+$ npm install @cosmwasm/cli --save-dev
+$ ./node_modules/.bin/cosmwasm-cli
+```
+
+### globally with yarn
+
+```
+$ yarn global add @cosmwasm/cli
+$ cosmwasm-cli
+```
+
+### globally with npm
+
+```
+$ npm install -g @cosmwasm/cli
+$ cosmwasm-cli
+```
+
+## Getting started
+
+1. Install `@cosmwasm/cli` and run `cosmwasm-cli` as shown above
+2. Start a local wasmd blockchain
+3. Start with `./bin/cosmwasm-cli --init examples/local_faucet.ts`
```

Comment:
Ah damn, this can only work from within the development repo. Otherwise the "helper" file is not very helpful. This whole CLI idea is a mess .... 🤯

Reply:
It feels like you can only lose when you try to balance between the necessary boilerplate to achive anything on the one side and flexibility of an API on the other side.

Reply:
Okay, I got it. This is a good place to play for now. 

For later, we can:

a. provide an example downloading the init file via `curl` with some sample setups
b. provide a second package (built into the binary) that provides a higher-level, opinionated API for common demo tasks, but doesn't create the objects